### ISSUE TITLE
Ability to add groups of groups

### ISSUE THREAD

**Issue by lemoncolly** [vincep5/MMM-MyStandings#1]
Due to the number of teams when showing all divisions, it would be useful to have groups of groups for each sport so it could show multiple divisions (but not all) at once on screen.

**Comment by vincep5** [comment]
Good idea, I will keep this open in the case that I want to tackle this idea when I have time.  Also, if anyone wants to contribute to the code.

**Comment by nishmeht7** [comment]
Hey @vincep5 - didn't know how to else to reach out, but I want to build and sell magic mirrors for people and would like to provide them with the option of adding this module. Is there an email I can I reach out to you at, to talk more about this?

**Comment by dathbe** [comment]
> Due to the number of teams when showing all divisions, it would be useful to have groups of groups for each sport so it could show multiple divisions (but not all) at once on screen.

Can you be more specific of what you're looking for here?  I might add it to my list of to-dos for my updated fork here:
https://github.com/dathbe/MMM-MyStandings
